data/transition-sites/gds_alphagov_alert.yml

```diff
@@ -0,0 +1,10 @@
+---
+site: gds_alphagov_alert
+whitehall_slug: government-digital-service
+homepage: https://alert.publishing.service.gov.uk/
```

Comment:
It would be nice to not have the trailing slashes on the homepage and global redirect URLs - sorry I didn't notice them before.